src/root/outbuffer.d

```diff
@@ -91,6 +91,16 @@ struct OutBuffer
         write(string, strlen(string));
     }
 
+    void writestring(const(char)[] s) nothrow
+    {
+        write(s.ptr, s.length);
+    }
+
+    void writestring(string s) nothrow
```

Comment:
AFAICS `write` is not holding any reference to `s`, so this overload isn't needed.